app/controllers/subscriptions_management_controller.rb

```diff
@@ -0,0 +1,149 @@
+class SubscriptionsManagementController < ApplicationController
+  before_action :set_cache_control_header
+  before_action :check_if_access_is_enabled
+  before_action :get_subscriptions
+
+  MISSING_EMAIL_ERROR = "Please enter your email address.".freeze
+  INVALID_EMAIL_ERROR = "This doesn’t look like a valid email address – check you’ve entered it correctly.".freeze
+
+  def index; end
+
+  def update_frequency
+    @back_url = list_subscriptions_path
```

Comment:
Is it worth setting this in a `before_action` or are we anticipating different `back_urls` depending on the action?

Reply:
They're all the same so I've changed it to a `before_action`